opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp

```diff
@@ -1139,6 +1147,9 @@ runSubStep_()
     catch (const TooManyIterations& e) {
         handleFailure("Solver convergence failure - Iteration limit reached", e);
     }
+    catch (const TimeSteppingBreakdown& e) {
+        handleFailure("Time step was too large", e);
```

Comment:
This message is misleading, see the other comment. Maybe pass e.what() as the first string, if the message is modified as suggested above (it will be repeated in the DBG file, but show up once on the terminal).